Plist2swift/Plist2swift.swift

```diff
@@ -320,6 +423,35 @@ extension String {
 	}
 }
 
+// MARK: - Tuples
+
+class KeyValueTuples {
+
+	var tuples: [(key: String, value: AnyObject)]
```

Comment:
please change this to Any:

```suggestion
	var tuples: [(key: String, value: Any)]
```